### PR TITLE
Update documentation for enhanced notification scripts

### PR DESCRIPTION
**Pull request by sysadmama**
Some additions and an argument overview. Hope that helps :)

refs #5170

### CODE REVIEW

File: doc/3-monitoring-basics.md [hunk 1]
```diff
@@ -1559,66 +1559,81 @@ interfaces (email, XMPP, IRC, Twitter, etc.).
...
-> other MTAs.
+> This example requires the `mail` binary installed on the Icinga 2
+> master.
+
+#### <a id="mail-host-notification"></a> mail-host-notification
+
+A quick overview of the arguments that can be used, mandatory ones are
+marked with an `x`. See also [host runtime
+macros](3-monitoring-basics.md#-host-runtime-macros) for further
+information.
```
Comment (dnsmichi) [inline doc/3-monitoring-basics.md]: This is listed in the `NotificationCommand` section, but describes the script arguments. Better would be to just list the custom attributes in the first column, and use the same description type including **Required.** or **Optional.** as we do anywhere else in tables.

**Comment by sysadmama** [comment]
Oh sorry. Didn't know my changes would be published immediately oO *hiding face*

**Comment by dnsmichi** [comment]
No worries. I will squash your changes after review, thanks :)

**Comment by dnsmichi** [comment]
### Final Review

- Rebased against master
- Squashed the commits into a single one

```
git checkout master
git remote add sysadma https://github.com/sysadmama/icinga2.git
git fetch --all
git checkout sysadmama/doc/notification-5273 -b doc/notification-5273
git log
git rebase -i HEAD~7
:s/pick/squash/
first line: pick
```

- Minor change with `Defaults to ...``, amended
- Minor formatting and URLs, amended
- `git push -f` to update the remote branch for this PR


Thanks a lot for your contribution 👍

**Pull request merged**